ui/apos/components/AposImportModal.vue

```diff
@@ -179,6 +185,21 @@ export default {
         this.updateProcess(notifId, processed, total);
       };
     },
+    async upload(notifId) {
```

Comment:
Thanks. I didn't realize this was not related to the PR in some way. I think it's easiest to maintain this way though.